pkg/rungroup/rungroup.go

```diff
@@ -0,0 +1,85 @@
+package rungroup
```

Comment:
Ah. So this is a _replacement_ for oklog's rungroup, not simple a wrapper. Seems legit